packages/ui/src/components/va-navbar/_variables.scss

```diff
@@ -2,7 +2,9 @@
   --va-navbar-height: 4.0625rem;
   --va-navbar-padding-left: 1rem;
   --va-navbar-padding-right: 1rem;
-  --va-navbar-transition: background-color 0.3s ease; // sidebar's bg color transitions as well -> consistency
+  --va-navbar-transition: all 0.3s ease;
+  --va-navbar-position: relative;
```

Comment:
Instead of using the relative, I'll suggest you to use the `static` value, because top, right, left and bottom properties won't effect on the HTML-element.

Reply:
Maxim said that `AppBar` & `NavBar` must be `relative` as default for some reasons.